admin/go.mod

```diff
@@ -0,0 +1,3 @@
+module galene/admin
+
+go 1.13
```

Comment:
Maybe I'm totally wrong, but I don't think that's necessary to define the go module if this admin interface ends up in Galène source.